attestation-agent/lib/src/config.rs

```diff
@@ -4,60 +4,50 @@
 //
 
 use anyhow::{anyhow, Context, Result};
-use kbs_protocol::{evidence_provider::NativeEvidenceProvider, KbsClientBuilder};
 use log::debug;
-use serde::{Deserialize, Serialize};
+use serde::Deserialize;
 use std::env;
+use std::fs::File;
 use std::path::Path;
 use std::sync::OnceLock;
 use tokio::fs;
 
 const PEER_POD_CONFIG_PATH: &str = "/run/peerpod/daemon.json";
+pub const DEFAULT_AA_CONFIG_PATH: &str = "/etc/attestation.toml";
```

Comment:
Maybe `attestation-agent.toml`? 

Also there was once a discussion with @danmihai1 , he suggested to use JSONs for readibility in initdata. I prefer TOML as we can add comments but JSONs cannot.

Reply:
Since the primary PR is about supporting attestation token, I think the changes to add config file mechanism should be in a separate PR.

This will help with the discussions and review especially w.r.to format, name and extensibility

cc @Xynnn007 @fitzthum